### DISCUSSION TITLE
The models trained ignore the prompt

### DISCUSSION THREAD

**Discussion by ivaxsirc** [d8ahazard/sd_dreambooth_extension#179] [General]
I have trained dreambooth with photos of my face, I put the instance prompt and the class prompt, and I have trained with 1e-6, 2e-6, 5e-6 and the same thing always happens, it ignores the prompt when the model loads.

For example, you write the instance prompt followed by, for example: "as a Greek statue", it doesn't matter. Only the trained photos is showing without any modification, when you remove the instance prompt from the prompt works.

I have tried with old models that I generate with WSL and works with the photos trained with the same automatic1111, my face trained as a Greek statue without problems

Does anyone else have this problem?
Thanks

**Comment by ivaxsirc** [comment]
I find, is the option Train text encoder, if you deselected works fine.

**Comment by NeptuneTQ** [comment]
The Text encoder makes the training a lot more efficient, so you don't need nearly as many steps, so you were probably over training the model.  In my experience,  I use to use 3500 steps for training but with text encoder on, only 600-1200 steps max.  I prefer the text encoder off however for faces as I feel I get less accurate models with it.

**Comment by 0xItx** [comment]
Disabling text encoder training along with using a low LR (1e-6) gives me decent results, though it takes a crazy amount of steps for my model (around 20-40k).

**Comment by ivaxsirc** [comment]
With the concept list things change, when you put two concepts you need the training text encoder for better results and not ignores the prompt
